roles/enable-fips/tasks/reboot.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Restart host"
+  shell: 'sleep 2 && shutdown -r now'
+  async: 1
+  poll: 0
+  ignore_errors: true
+
+- name: "Wait for reboot"
+  become: false
+  local_action: wait_for host={{ inventory_hostname }} port=22 state=started delay=10
```

Comment:
I would recommend doing this in the normal style:

```
local_action:
  wait_for:
    host:
```

Reply:
https://github.com/SatelliteQE/ansible-satellite6/blob/master/CONTRIBUTING.md#some-best-practices


Use Multi Line YAML notation, instead of key=value